docker-compose.yml

```diff
@@ -2,9 +2,12 @@ version: '2.4'
 
 x-web-defaults: &web-defaults
   restart: unless-stopped
+  image: ghcr.io/apexarray/obico-server/web:dev-django-4.2
   build:
     context: backend
     dockerfile: 'Dockerfile'
+    args:
+      BASE_IMAGE: ghcr.io/apexarray/obico-server/web-base:dev-django-4.2
```

Comment:
2 comments here:

1. How can I build the base image? I'd like to host the base image in the repo of official obico account.
2. "dev-django-4.2"  sounds like it's for development env, not for real deployment?

Reply:
reverted in [fad44bb](https://github.com/TheSpaghettiDetective/obico-server/pull/829/commits/fad44bb4f22e44437cf37544be1198f1f35933ec)